docs/basic_training/index.fr.md

```diff
@@ -0,0 +1,31 @@
+---
+description: Aperçu du matériel de formation Nextflow de base
+cacher :
```

Comment:
```suggestion
hide:
```

Reply:
Thanks again @mribeirodantas , I've accepted the suggestion.